homeassistant/components/lametric/services.py

```diff
@@ -113,12 +112,12 @@ async def _async_service_message(call: ServiceCall) -> None:
 async def async_send_notification(
     coordinator: LaMetricDataUpdateCoordinator,
     call: ServiceCall,
-    frames: Sequence[Chart | Simple],
+    frames: list[Chart | Goal | Simple],
```

Comment:
Frames is passed to `Model`. Since `list` is invariant, the type needs to include `Goal` to be compatible.

https://github.com/frenck/python-demetriek/blob/v0.4.0/src/demetriek/models.py#L165-L168